### PR TITLE
Add ILIKE to the list of comparators

### PR DESCRIPTION
**Pull request by svanharmelen**

### CODE REVIEW

File: components/ResultsTable.go [hunk 1]
```diff
@@ -1220,7 +1220,7 @@ func (table *ResultsTable) search() {
 
 	table.Filter.Input.SetAutocompleteFunc(func(currentText string) []string {
 		split := strings.Split(currentText, " ")
-		comparators := []string{"=", "!=", ">", "<", ">=", "<=", "LIKE", "NOT LIKE", "IN", "NOT IN", "IS", "IS NOT", "BETWEEN", "NOT BETWEEN"}
+		comparators := []string{"=", "!=", ">", "<", ">=", "<=", "LIKE", "ILIKE", "NOT LIKE", "IN", "NOT IN", "IS", "IS NOT", "BETWEEN", "NOT BETWEEN"}
```
Comment (ccoVeille) [inline components/ResultsTable.go:1223]: I'm surprised do not see REGEXP and NOT REGEXP

Comment (ccoVeille) [inline components/ResultsTable.go:1223]: Why not?


```suggestion
		comparators := []string{"=", "!=", ">", "<", ">=", "<=", "LIKE", "NOT LIKE", "ILIKE", "NOT ILIKE", "IN", "NOT IN", "IS", "IS NOT", "BETWEEN", "NOT BETWEEN"}
```

Comment (svanharmelen) [reply components/ResultsTable.go]: Sure, why not 😉

Comment (svanharmelen) [reply components/ResultsTable.go]: Added them as well while I was at it...

**Comment by svanharmelen** [comment]
I just added a few more missing components and improved the completion a little so it actually filters out suggestions that no longer match what a user has typed in. In addition ordered the suggestions for readability.

Will make a separate PR for fixing the lint error. Guess the linter is updated as this PR doesn't touch that code.

**Pull request merged**